cmd/app/options/app.go

```diff
@@ -81,3 +87,8 @@ func (e *ExtraHeaderOptions) AddFlags(fs *pflag.FlagSet) {
 			"proxied requests as part of the impersonated request. A single key can "+
 			"hold multiple values.")
 }
+
+func (k *Cluster) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&k.Config, "clusters-config", k.Config,
+		"Path to the YAML file containing an array of clusters. Each cluster in the array includes two fields: name and kubeconfig.")
```

Comment:
@chintansakhiya Will that yaml file contain kubeconfig or path of kubeconfig?

Reply:
yaml file contain path of kubeconfig